app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -95,7 +101,13 @@ class PatientSummaryScreen(context: Context, attrs: AttributeSet) : RelativeLayo
 
     setupSummaryList()
 
-    Observable.mergeArray(screenCreates(), backClicks(), doneClicks(), adapterUiEvents)
+    Observable.mergeArray(
+        screenCreates(),
+        backClicks(),
+        doneClicks(),
+        adapterUiEvents,
+        bloodPressureSaves(),
+        appointmentScheduleSheetClosed())
         .observeOn(io())
         .compose(controller)
         .observeOn(mainThread())
```

Comment:
Formatting: the `mergeArray()` operators have the same indentation as `observeOn()` and other operators, making it look like they're at the same nesting level. 

One trick I use is to move the static function in a new line too, making the nesting clear:

```kotlin
Observable
    .mergeArray(
        screenCreates(),
        backClicks(),
        doneClicks(),
        adapterUiEvents,
        bloodPressureSaves(),
        appointmentScheduleSheetClosed())
    .observeOn(io())
    .compose(controller)
    .observeOn(mainThread())
    .takeUntil(RxView.detaches(this))
    .subscribe { uiChange -> uiChange(this) }
```

It's okay if you want to change this in your next PR.